.github/workflows/actions-integration-tests.yml

```
@@ -0,0 +1,89 @@
+
+      - name: Run migrations
+        working-directory: actions-indexer-repository
+        run: sqlx migrate run --source src/postgres/migrations
+
+      - name: Generate sqlx query cache
+        working-directory: actions-indexer-repository
+        run: cargo sqlx prepare
+
+      - name: Run integration tests
+        env:
+          RUST_BACKTRACE: 1
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)